.github/workflows/run-k8s-tests-on-aks.yaml

```diff
@@ -46,7 +51,8 @@ jobs:
             -n "${{ github.event.pull_request.number }}-${{ github.event.pull_request.head.sha }}-${{ matrix.vmm }}-amd64" \
             -s "Standard_D4s_v5" \
             --node-count 1 \
-            --generate-ssh-keys
+            --generate-ssh-keys \
+            ${{ matrix.host_os == 'cbl-mariner' && '--os-sku mariner --workload-runtime KataMshvVmIsolation' }}
```

Comment:
is mariner == cbl-mariner? or are we talking about different flavors?

I am asking because I have another suggestion. Rename `host_os` from cbl-mariner to mariner on line 27. Here we simply pass `--os-sku ${{ matrix.host_os }}` (I assume that `--os-sku ubuntu` is valid). This way adding a new host OS is easy. Then one extra line will be needed for passing the `--workload-runtime`.

Reply:
Yes, Mariner is just a shorthand for CBL-Mariner. I use `cbl-mariner` to match #6465 and because I'd find it confusing to refer to both `cbl-mariner` and `mariner` in the codebase. This way we can simply pass down the `host_os` value to determine which image/initrd to build.